concepts/05 UI Components/TreeList/10 Columns/50 Adaptability/05 Hide columns.md

```diff
@@ -0,0 +1,103 @@
+TreeList can hide columns to adapt to screens of different sizes. To add this feature, set the [columnHidingEnabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#columnHidingEnabled) to **true**.
+
+Each column has a unique hiding priority. The rightmost column is the base column with the priority of 0. This value is incremented by 1 for columns from right to left; column with the lowest priority is hidden first.
```

Comment:
- Where does the term "base column" come from?
- Only in the last sentence we give a hint that hiding priority participates in determining the order in which the columns are hidden.

Reply:
Each column has a unique _**hiding priority**_. "Hiding priority" speaks for itself, i.e. it sets the hiding priority (order)?